tests/plugins/fake-governance-plugin/src/main/java/ce/ws/FakeGovWs.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package ce.ws;
+
+import java.util.Arrays;
+import org.sonar.api.server.ws.WebService;
+
+public class FakeGovWs implements WebService {
+  private final FakeGoVWsAction[] actions;
+
+  public FakeGovWs(FakeGoVWsAction[] actions) {
+    this.actions = actions;
+  }
+
+  @Override
+  public void define(Context context) {
+    System.err.println("Fake GOV WS being setup: " + actions.length);
```

Comment:
Shouldn't this be in a Logger (so we may see it in web.log during a test ?)

Reply:
this should just be removed